speech/google/cloud/speech/_gax.py

```diff
@@ -72,9 +88,26 @@ def async_recognize(self, sample, language_code=None,
                                and phrases. This can also be used to add new
                                words to the vocabulary of the recognizer.
 
-        :raises NotImplementedError: Always.
+        :rtype: :class:`~google.cloud.operation.Opeartion`
+        :returns: Instance of ``Operation`` to poll for results.
         """
-        raise NotImplementedError
+        config = RecognitionConfig(
+            encoding=sample.encoding, sample_rate=sample.sample_rate,
+            language_code=language_code, max_alternatives=max_alternatives,
+            profanity_filter=profanity_filter,
+            speech_context=SpeechContext(phrases=speech_context))
+
+        audio = RecognitionAudio(content=sample.content,
+                                 uri=sample.source_uri)
+        api = self._gapic_api
+        response = api.async_recognize(config=config, audio=audio)
+
+        self._client._operations_stub = make_secure_stub(
```

Comment:
This shouldn't happen here. Instead do it in the constructor for `GAPICSpeechAPI`. Technically you don't need to pass a client to `Operation.from_pb()`.

If `self._operations_stub` was defined (in the constructor) for this `GAPICSpeechAPI` object then you could just use

```python
Operation.from_pb(response, self)
```